slothy/targets/aarch64/aarch64_neon.py

```diff
@@ -3014,6 +3014,37 @@ def make(cls, src):
             ]
         return obj
 
+class St3(AArch64Instruction): # pylint: disable=missing-docstring,invalid-name
+    pass
+
+class st3_base(St3): # pylint: disable=missing-docstring,invalid-name
+    pattern = "st3 {<Va>.<dt0>, <Vb>.<dt1>, <Vc>.<dt2>}, [<Xc>]"
+    inputs = ["Xc", "Va", "Vb", "Vc"]
+    @classmethod
+    def make(cls, src):
+        obj = AArch64Instruction.build(cls, src)
+        obj.offset_adjustable = False
+        obj.addr = obj.args_in[0]
+        obj.args_in_combinations = [
+                ( [1,2,3], [ [ f"v{i}", f"v{i+1}", f"v{i+2}" ] for i in range(0,29) ] )
+            ]
+        return obj
+
+class st3_with_inc(St3): # pylint: disable=missing-docstring,invalid-name
+    pattern = "st3 {<Va>.<dt0>, <Vb>.<dt1>, <Vc>.<dt2>}, [<Xc>], <imm>"
+    inputs = ["Va", "Vb", "Vc"]
+    in_outs = ["Xc"]
+    @classmethod
+    def make(cls, src):
+        obj = AArch64Instruction.build(cls, src)
+        obj.addr = obj.args_in_out[0]
+        obj.increment = obj.immediate
+        obj.pre_index = None
+        obj.args_in_combinations = [
+                ( [0,1,2], [ [ f"v{i}", f"v{i+1}", f"v{i+2}" ] for i in range(0,29) ] )
```

Comment:
Should this be `range(0,30)`? Similar Q for pre-existing models or 2/4-fold ld/st.

Reply:
you are right... they are all off by one... 
let me fix that.

Reply:
fixed.

Reply:
Thanks!